libs/core/src/stories/typography.docs.mdx

```diff
@@ -1,8 +1,24 @@
-
 import { Meta, Story } from '@storybook/blocks';
+import {DocTokenTable} from '@pine-ds/doc-components';
+
+<Meta title="Foundations/Design Tokens/Typography"/>
+
+# Font Family
+
+<DocTokenTable category="font-family" />
+
+# Font Size
+
+<DocTokenTable category="font-size" />
+
+# Font Weight
+
+<DocTokenTable category="font-weight" />
```

Comment:
For this and font family, the font family in storybook doesn't reflect what is expected of the design system because we never set it up that way. I feel like we should change this, but that feels like a team decision. What do you think?